concepts/numbers/about.md

```diff
@@ -1,8 +1,50 @@
 # About
 
-## Type Conversion
+Go contains basic numeric types that can represent sets of either integer or
```

Comment:
Feel free to disregard if you disagree, but I advocate putting line breaks at sentence breaks in this documentation. It line wraps both in the UI and in the reviewing UI, but breaking at sentences means that your changes to individual sentences end up having less of a blast radius.

The way it is currently broken down means that you have to reformat the whole paragraph whenever you edit one sentence (causing a larger blast radius and making it slightly harder to review).

As I said, feel free to disregard if you prefer this style though.

Reply:
I like the above suggestion.